common/djangoapps/course_modes/models.py

```diff
@@ -367,6 +380,43 @@ def min_course_price_for_currency(cls, course_id, currency):
         modes = cls.modes_for_course(course_id)
         return min(mode.min_price for mode in modes if mode.currency == currency)
 
+    @classmethod
+    def has_professional_mode(cls, modes_dict):
+        """
+        check the course mode is profession or no-id-professional
+
+        Args:
+            modes_dict (dict): course modes.
+
+        Returns:
+            bool
+        """
+        return "professional" in modes_dict or CourseMode.NO_ID_PROFESSIONAL_MODES[0] in modes_dict
+
+    @classmethod
+    def is_professional_mode(cls, course_mode_tuple):
+        """
+        checking that tuple is professional mode.
+        Args:
+            course_mode_tuple (tuple) : course mode tuple
+
+        Returns:
+            bool
+        """
+        if course_mode_tuple:
+            return course_mode_tuple.slug == 'professional' or course_mode_tuple.slug == cls.NO_ID_PROFESSIONAL_MODES[0]
```

Comment:
I think we can't replace the NO_ID_PROFESSIONAL_MODES variable with PROFESSIONAL_MODES because if we do that we can't distinguish if professional mode require the identification or not and also the 'professional' mode is also in VERIFIED_MODES list. We can make a new list PROFESSIONAL_MODES = ["professional", "no-id-professional"] for this purpose keeping the old list as it.